app/models/site.rb

```diff
@@ -56,6 +56,8 @@ class Site < ApplicationRecord
 
   after_commit -> { GeocodeJob.perform_later("Site", id) }, if: :needs_geolocation?
 
+  scope :with_locatable_address, -> { where.not(address1: "").or(Site.unscoped.where.not(postcode: "")) }
```

Comment:
```suggestion
  scope :with_locatable_address, -> { where.not(address1: "", postcode: "")) }
```

Can we keep it tight
https://github.com/DFE-Digital/teacher-training-api/blob/f41b6f576ac1225a4c00fb617a5e46670ad2e998/app/models/site.rb#L44-L47

Reply:
I think that would do an `AND` and we need an `OR` here. I'm closing this PR for now anyway for a rejig.